docs/overview/excel.md

```diff
@@ -20,10 +20,10 @@ This series of documents teaches you how to use these tools. You'll be introduce
 
 To use Office Scripts, you'll need the following.
 
-1. [Excel on the web](https://www.office.com/launch/excel) (Excel on Windows can only use Office Scripts with [script buttons](../develop/script-buttons.md)).
+1. Excel for Windows or Excel on the web.
 
     > [!TIP]
-    > Office Scripts are now available in Office on Windows and on Mac for [Office Insiders](https://insider.office.com/).
+    > Office Scripts are now available in Office on Mac for [Office Insiders](https://insider.office.com/).
```

Comment:
We might be able to remove this - the plan is to roll out Mac and Windows to GA around the same time